spec/systems/search_keywords_spec.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'search keywords', type: :system do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
+
+  context 'given exactly match search keyword' do
+    it 'displays searched keywords' do
+      user = Fabricate(:user)
+      file_path = Rails.root.join('spec', 'fabricators', 'files', 'adword_keywords.csv')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
+
+        click_button(I18n.t('auth.login'))
+      end
+
+      attach_file('csv_import_form[file]', file_path)
```

Comment:
No need to do the upload in this test as that was covered by another test already.

In this test, we can create the Keyword directly to the database

```
Keyword.create(xxxx)

With Fabricate we can do

Fabricate(:keyword, xxx) as I remember
```

Reply:
Fixed in d789b70f7d8603a2f2b11b6379d2ddff73a5a8b1